portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,23 +23,40 @@
 import javax.servlet.http.HttpServletResponse;
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> fruits;
+  private ArrayList<String> comments;
+
 
   @Override
   public void init(){
-    fruits = new ArrayList<String>();
-    fruits.add("Blueberry");
-    fruits.add("Cherry");
-    fruits.add("Peach");
-    fruits.add("Apricot");
-    fruits.add("Mango");
+    comments = new ArrayList<String>();    
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Gson gson = new Gson();
-    String jsonfruit = gson.toJson(fruits);
     response.setContentType("application/json;");
-    response.getWriter().println(jsonfruit);
+    String json = new Gson().toJson(comments);
+    response.getWriter().println(json);
+  }
+
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I think this should have an @Override tag?

Reply:
It definitely should! Thanks for catching that